audio/beets/Portfile

```diff
@@ -83,7 +83,8 @@ if {${name} eq ${subport} || "${name}-devel" eq ${subport}} {
                     port:py${python.version}-yaml
 
     if {${name} eq ${subport}} {
-        depends_lib-append \
+        depends_lib-delete \
+                    port:py${python.version}-gmusicapi \
                     port:py${python.version}-six
```

Comment:
I don't see `py-six` in the earlier dependency list. Is this supposed to be an add, or...?

Reply:
As for the dependency logic in-general, it would be preferable to invert it: Only add deps that are shared between both, and then selectively add others for the Devel port:

```
if {${name} ne ${subport}} {
    depends_lib-append \
        port:py${python.version}-gmusicapi
}
```

Does that make sense?

Reply:
@mascguy it was removed from the last update. I've introduced a typo with `-append` instead of `-delete` before.

Reply:
Hm, it should be here. Let me double check it.

Reply:
@mascguy you are defently right. Thanks to catch this issue!

Reply:
@mascguy to make things clear:
 - the last release required both `py-gmusicapi` and `py-six`;
 - the current master is removed this dependency.
 
 I've removed `py-six` at b6fb1fd0e12d3bfb27dd4ccd501233d5ec1a4b42 but here introduced a very bad typo.